aiohomekit/controller/ip/connection.py

```diff
@@ -362,7 +365,10 @@ async def post(self, target, body, content_type=HttpContentTypes.TLV):
         return await self.request(
             method="POST",
             target=target,
-            headers=[("Content-Length", len(body)), ("Content-Type", content_type)],
+            headers=[
+                ("Content-Length", len(body)),
+                ("Content-Type", content_type.value),
+            ],
             body=body,
```

Comment:
```
    async def post_json(self, target, body):                                                                                                                            
        response = await self.post(                                                                                                                                     
            target,                                                                                                                                                     
            hkjson.dump_bytes(body),                                                                                                                                    
            content_type=HttpContentTypes.JSON,                                                                                                                         
        )     
```

probably needs more of the same ?

Reply:
```
    async def post_tlv(self, target, body, expected=None):
        try:
            response = await self.post(
                target,
                TLV.encode_list(body),
                content_type=HttpContentTypes.TLV,
            )
```

Reply:
nm, its a wrapper around what you changed 👍 
haha... commenting before coffee

Reply:
Both are fine without - it's calling `self.post`, and that "unpacks" the enum into a string.